src/Entities/IssueDetails.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ITBugTracking\Entities;
+
+use ITBugTracking\Entities\Issue;
+
+class IssueDetails extends Issue
+{
+    public array|null $comments;
```

Comment:
I think the correct value for an issue with no comments is `[]` (empty array), not `null`.

Reply:
done

Reply:
This type hint still looks like null is an option to me though?

Reply:
OK, it behaving now...

Reply:
*it's*